frontend/entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-envsubst '${FRONTEND_PORT} ${VITE_USER_SERVICE} ${VITE_QUESTION_SERVICE} ${VITE_MATCHING_SERVICE}' < /etc/nginx/nginx.conf.template > /etc/nginx/conf.d/default.conf
+envsubst '${FRONTEND_PORT} ${VITE_USER_SERVICE} ${VITE_QUESTION_SERVICE} ${VITE_COLLAB_SERVICE} ${VITE_COLLAB_WS} ${VITE_MATCHING}' < /etc/nginx/nginx.conf.template > /etc/nginx/conf.d/default.conf
```

Comment:
Should be `VITE_MATCHING_SERVICE` as in the nginx conf template

Reply:
Resolved!